kubelet/README.md

```diff
@@ -56,6 +56,21 @@ The check will still be able to collect:
 
 See [service_checks.json][6] for a list of service checks provided by this integration.
 
+### Excluded containers
+
+It is possible to restrict the data collected to a subset of the containers
+deployed. This is done setting the [`DD_CONTAINER_EXCLUDE` environment
+variable][7]. This integration does not report metrics from the containers
+specified in that environment variable.
+
+This integration also reports network metrics that apply to a pod instead of a
+single container. In that case, if `DD_CONTAINER_EXCLUDE` applies to a
```

Comment:
```suggestion
You can restrict the data collected to a subset of the containers
deployed by setting the [`DD_CONTAINER_EXCLUDE` environment
variable][7]. This integration does not report metrics from the containers
specified in that environment variable.

This integration reports network metrics that apply to a pod instead of a
single container. So, if `DD_CONTAINER_EXCLUDE` applies to a
```